### PR TITLE
use resources of cpp-tests in js-tests

### PR DESCRIPTION
**Pull request by zhaijialong**

### CODE REVIEW

File: build/cocos2d_tests.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -774,6 +774,76 @@
...
+		42AE838F1A89A275009476C9 /* background.ogg in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CAD18CED84500F37B72 /* background.ogg */; };
+		42AE83901A89A275009476C9 /* effect2.ogg in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CB718CED84500F37B72 /* effect2.ogg */; };
+		42AE83911A89A275009476C9 /* fileLookup.plist in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CB918CED84500F37B72 /* fileLookup.plist */; };
+		42AE83921A89A275009476C9 /* Hello.png in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CBD18CED84500F37B72 /* Hello.png */; };
+		42AE83931A89A275009476C9 /* effect1.raw in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CB518CED84500F37B72 /* effect1.raw */; };
+		42AE83941A89A275009476C9 /* effect1.wav in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CB618CED84500F37B72 /* effect1.wav */; };
+		42AE83951A89A275009476C9 /* pew-pew-lei.wav in CopyFiles */ = {isa = PBXBuildFile; fileRef = 1AC35CC418CED84500F37B72 /* pew-pew-lei.wav */; };
+		42AE83961A89A275009476C9 /* Test.html in CopyFiles */ = {isa = PBXBuildFile; fileRef = 29AFEF6619ACCAA000F6B10A /* Test.html */; };
```
Comment (minggo) [inline build/cocos2d_tests.xcodeproj/project.pbxproj]: It is better to add a command to copy resources as lua did.

File: build/cocos2d-win32.vc2012.sln [hunk 2]
```diff
@@ -83,12 +80,6 @@ Global
 		{929480E7-23C0-4DF6-8456-096D71547116}.Release|ARM.ActiveCfg = Release|Win32
 		{929480E7-23C0-4DF6-8456-096D71547116}.Release|Win32.ActiveCfg = Release|Win32
 		{929480E7-23C0-4DF6-8456-096D71547116}.Release|Win32.Build.0 = Release|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Debug|ARM.ActiveCfg = Debug|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Debug|Win32.ActiveCfg = Debug|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Debug|Win32.Build.0 = Debug|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Release|ARM.ActiveCfg = Release|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Release|Win32.ActiveCfg = Release|Win32
-		{1DB7C0FC-46FF-4A1B-82E0-C6244EEEC4C2}.Release|Win32.Build.0 = Release|Win32
```
Comment (minggo) [inline build/cocos2d-win32.vc2012.sln]: We removed VS2012 support in @10418.
So should synchronize it.

File: build/cocos2d_tests.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -6780,7 +6967,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(SRCROOT)/../plugin/plugins/facebook/proj.ios/sdk",
+					"$(SRCROOT)/../plugin/plugins/facebook/proj.ios/sdk/**",
```
Comment (minggo) [inline build/cocos2d_tests.xcodeproj/project.pbxproj:6599]: Why change it?
Does it break other tests?

File: tests/cpp-tests/Resources/Shaders/example_outline.fsh [hunk 4]
```diff
@@ -25,7 +25,7 @@ void main()
     
     accum *= u_threshold;
     accum.rgb =  u_outlineColor * accum.a;
-    accum.a = 1.0;
+    //accum.a = 1.0; why set to 1.0 ?
```
Comment (minggo) [inline tests/cpp-tests/Resources/Shaders/example_outline.fsh]: Could you please modify it in another pull request.
We can discuss it there.
It is better to do one thing for a pull request.

File: tests/js-tests/res/Shaders/example_Twist.vsh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+// http://www.cocos2d-iphone.org
+
+attribute vec4 a_position;
+
+void main()
+{
+    gl_Position = CC_MVPMatrix * a_position;
+}
```
Comment (minggo) [inline tests/js-tests/res/Shaders/example_Twist.vsh:8]: I think cpp-tests have these shader files, right?

File: tests/cpp-tests/Resources/Shaders/example_outline.fsh (hunk 4 above)
Comment (zhaijialong) [inline tests/cpp-tests/Resources/Shaders/example_outline.fsh]: ok, no problem

File: build/cocos2d_tests.xcodeproj/project.pbxproj (hunk 3 above)
Comment (zhaijialong) [inline build/cocos2d_tests.xcodeproj/project.pbxproj:6599]: It doesn't break other tests.

File: tests/js-tests/res/Shaders/example_Twist.vsh (hunk 5 above)
Comment (zhaijialong) [inline tests/js-tests/res/Shaders/example_Twist.vsh:8]: cpp-tests doesn't have these shader files, though it has shader file which has the same content but different file names.

File: build/cocos2d_tests.xcodeproj/project.pbxproj (hunk 3 above)
Comment (minggo) [inline build/cocos2d_tests.xcodeproj/project.pbxproj:6599]: Ok.
Thanks.

File: tests/js-tests/res/Shaders/example_Twist.vsh (hunk 5 above)
Comment (minggo) [inline tests/js-tests/res/Shaders/example_Twist.vsh:8]: Ok.
These shaders seems meaningless.
I think we should remove them, and add more meaningful shaders.

**Pull request merged**